cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -25,10 +25,10 @@ spec:
           # keep request = limit to keep this container in guaranteed class
```

Comment:
Now that memory request and limit are different, that comment is probably wrong.